[0.5][3.1.190618.1] - Reset settings - white screen

**Describe the bug**
White screen occurred after 'Reset Settings'.

**To Reproduce**
Steps to reproduce the behavior:
1. Go to 'Settings' - 'Reset settings' in Light Client or [staging.bitshares.org/#/settings/reset](https://staging.bitshares.org/#/settings/reset)
2. Click on 'Reset settings'
3. See white screen

**Expected behavior**
Don`t see a white screen

**Screenshots**
Light Client:
![MxKlbNGsBX](https://user-images.githubusercontent.com/17462020/60617468-7ef71a00-9ddc-11e9-9af9-38abea7c6a23.gif)
Firefox:
![sFtHyOjfcm](https://user-images.githubusercontent.com/17462020/60617477-83233780-9ddc-11e9-8154-4cfad5331bf1.gif)

**Desktop (please complete the following information):**
 - Windows 10
 - Firefox 67.0.4
 - Light Client 3.1.190618.1

**Additional context**
```
app.d25b9ac53f3ceac25bbf.js:1 Error: Minified React error #31; visit https://reactjs.org/docs/error-decoder.html?invariant=31&args[]=object%20with%20keys%20%7Btranslate%7D&args[]= for the full message or use the non-minified dev environment for full errors and additional helpful warnings. 
    at e.exports (vendor.5ca2cb1ba27b2b66d5c2.js:148)
    at p (vendor.5ca2cb1ba27b2b66d5c2.js:148)
    at di (vendor.5ca2cb1ba27b2b66d5c2.js:148)
    at vendor.5ca2cb1ba27b2b66d5c2.js:148
    at ki (vendor.5ca2cb1ba27b2b66d5c2.js:148)
    at xi (vendor.5ca2cb1ba27b2b66d5c2.js:148)
    at Ni (vendor.5ca2cb1ba27b2b66d5c2.js:148)
    at da (vendor.5ca2cb1ba27b2b66d5c2.js:148)
    at ha (vendor.5ca2cb1ba27b2b66d5c2.js:148)
    at Ga (vendor.5ca2cb1ba27b2b66d5c2.js:148)

app.d25b9ac53f3ceac25bbf.js:1 BindToChainState(t) Error: Minified React error #31; visit https://reactjs.org/docs/error-decoder.html?invariant=31&args[]=object%20with%20keys%20%7Btranslate%7D&args[]= for the full message or use the non-minified dev environment for full errors and additional helpful warnings. 
    at e.exports (vendor.5ca2cb1ba27b2b66d5c2.js:148)
    at p (vendor.5ca2cb1ba27b2b66d5c2.js:148)
    at di (vendor.5ca2cb1ba27b2b66d5c2.js:148)
    at vendor.5ca2cb1ba27b2b66d5c2.js:148
    at ki (vendor.5ca2cb1ba27b2b66d5c2.js:148)
    at xi (vendor.5ca2cb1ba27b2b66d5c2.js:148)
    at Ni (vendor.5ca2cb1ba27b2b66d5c2.js:148)
    at da (vendor.5ca2cb1ba27b2b66d5c2.js:148)
    at ha (vendor.5ca2cb1ba27b2b66d5c2.js:148)
    at Ga (vendor.5ca2cb1ba27b2b66d5c2.js:148)
```

--- a/app/components/Layout/Footer.jsx
+++ b/app/components/Layout/Footer.jsx
@@ -194,6 +194,9 @@ class Footer extends React.Component {
 
         const testNet = node.url.indexOf("testnet") !== -1;
 
+        if (!!node.location && node.location.translate)
+            node.location = counterpart.translate(node.location.translate);
+
         let title = node.operator + " " + !!node.location ? node.location : "";
         if ("country" in node) {
             title = node.country + (!!title ? " - " + title : "");
@@ -613,12 +616,11 @@ class Footer extends React.Component {
                                         }}
                                     >
                                         <div className="footer-status">
-                                            {connected &&
-                                                activeNode.testNet && (
-                                                    <span className="testnet">
-                                                        <Translate content="settings.testnet_nodes" />{" "}
-                                                    </span>
-                                                )}
+                                            {connected && activeNode.testNet && (
+                                                <span className="testnet">
+                                                    <Translate content="settings.testnet_nodes" />{" "}
+                                                </span>
+                                            )}
                                             {!connected ? (
                                                 <span className="warning">
                                                     <Translate content="footer.disconnected" />
@@ -638,10 +640,10 @@ class Footer extends React.Component {
                                                 {!connected
                                                     ? "-"
                                                     : !activeNode.ping
-                                                        ? "-"
-                                                        : parseInt(
-                                                              activeNode.ping
-                                                          ) + "ms"}
+                                                    ? "-"
+                                                    : parseInt(
+                                                          activeNode.ping
+                                                      ) + "ms"}
                                                 &nbsp;/&nbsp;
                                                 <span className="footer-block-title">
                                                     <Translate content="footer.block" />

--- a/app/components/Settings/AccessSettings.jsx
+++ b/app/components/Settings/AccessSettings.jsx
@@ -227,11 +227,9 @@ class ApiNode extends React.Component {
         let location =
             !!node.location &&
             typeof node.location === "object" &&
-            "translate" in node.location ? (
-                <Translate component="span" content={node.location.translate} />
-            ) : (
-                node.location
-            );
+            "translate" in node.location
+                ? counterpart.translate(node.location.translate)
+                : node.location;
 
         let title = !!location ? location : "";
         if (!!node.country) {
@@ -393,18 +391,13 @@ class AccessSettings extends React.Component {
     }
 
     _getConnectedNode() {
-        let connectedURL = null;
-        if (!this.props.connectedNode) {
-            connectedURL = autoSelectionUrl;
-        } else {
-            connectedURL = this.props.nodes.find(
-                node => node.url == this.props.connectedNode
-            );
-        }
-        if (!connectedURL) {
-            connectedURL = autoSelectionUrl;
-        }
-        return this.getNode(connectedURL);
+        let connectedURL = this.props.connectedNode || autoSelectionUrl;
+
+        const connectedNode = this.props.nodes.find(
+            node => node.url == connectedURL
+        );
+
+        return connectedNode ? this.getNode(connectedNode) : null;
     }
 
     _connectedNodeIsPersonal() {

--- a/app/components/Settings/ResetSettings.jsx
+++ b/app/components/Settings/ResetSettings.jsx
@@ -3,6 +3,7 @@ import counterpart from "counterpart";
 import Translate from "react-translate-component";
 import SettingsActions from "actions/SettingsActions";
 import {Button} from "bitshares-ui-style-guide";
+import willTransitionTo from "../../routerTransition";
 
 export default class ResetSettings extends React.Component {
     constructor() {
@@ -62,6 +63,9 @@ export default class ResetSettings extends React.Component {
                             this._setMessage(
                                 "settings.restore_default_success"
                             );
+                            setTimeout(() => {
+                                willTransitionTo(false);
+                            }, 50);
                         });
                     }}
                 >

